doc/jsk_perception/nodes/craft_node.rst

```diff
@@ -0,0 +1,82 @@
+craft_node.py
+=============
+
+What is this?
```

Comment:
How about writing the Quick Start in the comments of this PR in the README as well?

For example, the following information seems very important to me.
```
For python2 users, please do pip install pytesseract==0.3.1 manually because the latest version of pytesseract is not installed automatically.
```

In addition, I was confused when installing pytorch with python2.
The followint web page show only for python3 user.
https://pytorch.org/get-started/locally/
Could you please add pytorch installation process for python 2 users?

Reply:
Related to the these python version problem, how about using `catkin_virtualenv` to use the latest pip programs with Python3.x.
I think this makes the installation process easier.

Reply:
> Related to the these python version problem, how about using catkin_virtualenv to use the latest pip programs with Python3.x. I think this makes the installation process easier.

I'm not familiar with `catkin_virtualenv`. I have a question.
catkin_install_python calls catkin_install_python, but the scripts file is already written as a target in CMakelists.txt in jsk_perception. https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_perception/CMakeLists.txt#L430-L435
Does this mean that the scripts file also needs to be changed to work with python3?
If that is true, I think we need to address it with another PR.

Reply:
> How about writing the Quick Start in the comments of this PR in the README as well?

> Could you please add pytorch installation process for python 2 users?

Thanks. I added them.
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2650/commits/4a49d1d0668c58213e7c3d89b1d018bf7b082650
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2650/commits/a51b8ca6d6948c313150bb085c9d7a06301ccbb0

Reply:
> I'm not familiar with catkin_virtualenv. I have a question.
catkin_install_python needs to call catkin_install_python, but the script files are already written as a target in CMakelists.txt in jsk_perception. master/jsk_perception/CMakeLists.txt#L430-L435
Does this mean that the script files also need to be changed to work with python3?

I am very sorry, but you are right. We should not use `catkin_virtualenv`.

This process will also override the standard catkin_install_python macro to wrap a virtualenv loader around the specified python scripts.
https://github.com/locusrobotics/catkin_virtualenv

In addition to that, using `catkin_virtualenv` adds new dependency to `jsk_perception`. So we should not use `catkin_virtualenv`

Reply:
OK. I understand it.